doc/modules/language-guide/pages/modules-and-imports.adoc

```diff
@@ -91,19 +91,21 @@ The call to `+Counters.Counter(1)+` installs a fresh counter on the network. Ins
 
 The type annotation  `+: Counters.Counter+` is redundant here. It's included only to illustrate that the type of the actor class is available when required.
 
-== Importing from another canister
+== Importing from another canister smart contract
 
-In addition to the examples above that import {proglang} modules, you can also import functions from canisters by using the `+canister:+` prefix in place of the `+mo:+` prefix.
+In addition to the examples above that import {proglang} modules, you can also import actors (and their shared functions) from canister smart constracts by using the `+canister:+` prefix in place of the `+mo:+` prefix.
+
+NOTE: Unlike a Motoko library, an imported canister can be implemented in any other {IC} language that emits Candid interfaces for its canister smart contracts (for instance Rust), not just Motoko. It could even be an older or newer version of Motoko.
 
 For example, you might have a project that produces the following three canisters:
 
-* BigMap
-* Connectd
-* LinkedUp
+* BigMap (implemented in Rust)
+* Connectd (implemented in Motoko)
+* LinkedUp (implemented in Motoko)
 
 These three canisters are declared in the project's `+dfx.json+` configuration file and compiled by running `+dfx build+`.
 
-You can then use the following lines to import the functions from the `+BigMap+` and `+Connectd+` canisters in the LinkedUp program:
+You can then use the following lines to import the `+BigMap+` and `+Connectd+` canisters as actors in the Motoko LinkedUp actor:
```

Comment:
Do we define what we mean by "actor" anywhere in this documentation?

This sentence reads like all canisters are actors, or can be made actors if you use them right (e.g. you can import a canister implemented in AssemblyScript and use it as an actor). Not sure if that's the intentional reading.

Reply:
It is actually, although technically the canister would need to speak candid.

Reply:
```suggestion
You can then use the following lines to import the `+BigMap+` and `+Connectd+` canisters as actors in the {proglang} LinkedUp actor:
```

Reply:
```suggestion
You can then use the following lines to import the `+BigMap+` and `+Connectd+` canisters as actors in the {proglang} LinkedUp actor:
```